pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -1,4 +1,4 @@
-#hero
+#hero{onclick:"clickHero()"}
```

Comment:
I think it's generally considered a good idea to attach these event handlers in the javascript code itself, rather than using onclick attributes, since in theory the onclick attribute could get executed before the `clickHero()` function has been created, causing a runtime exception. i.e.

```js
$.ready(function() {
  $("#hero").click(clickHero);
});
```